compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -1,3 +1,7 @@
+hir_analysis_abi_custom_clothed_function =
+    functions with the `"custom"` ABI must be naked
+    .suggestion = add the `#[unsafe(naked)]` attribute to this function
```

Comment:
I think it's worth hinting at the other way to have a `custom` function. Not the best wording, but maybe something like:

```text
    items with the `"custom"` ABI can only be declared externally or defined via naked functions
    .suggestion = convert this to an `#[unsafe(naked)]` function
```